Script/AtomicNET/AtomicNETProject.json

```diff
@@ -23,6 +23,7 @@
             "name": "AtomicNET",
             "atomicNET" : true,
             "outputType" : "Library",
+            "defineConstants" : ["ATOMIC_PROFILING"],
```

Comment:
People would be profiling a release build anyhow, this flag missing on release builds would certainly cause some confusion.